bucket_54/ctwm/descriptions/desc.man

```diff
@@ -0,0 +1 @@
+Manual pages for ctwm.
```

Comment:
This file is optional.  If missing, one is created automatically (like plist.docs and plist.nls)